lib/improver/nowcasting/optical_flow.py

```diff
@@ -826,7 +814,7 @@ def process_dimensionless(self, data1, data2, xaxis, yaxis):
             self._zero_advection_velocities_warning(vel_comp, rain_mask)
         return ucomp, vcomp
 
-    def process(self, cube1, cube2):
+    def process(self, cube1, cube2, boxsize=30):
```

Comment:
It feels like this process method could be shortened a bit by making use of the [check_if_grid_is_equal_area](https://github.com/metoppv/improver/blob/master/lib/improver/utilities/spatial.py#L49), which might help given that there's quite a lot of checking that happens within this process method before the optical flow calculation actually begins.

Reply:
Ah thanks for this!  Will push up the simple stuff and take a look at this bit in the morning.

Reply:
Done.